src/portals/hope/features/claims/claim-details/ClaimMemberStatement.tsx

```diff
@@ -0,0 +1,313 @@
+import {
+  FadeIn,
+  Flex,
+  Input,
+  isPressing,
+  Keys,
+  Label,
+  Paragraph,
+  Shadowed,
+  usePlatform,
+} from '@hedvig-ui'
+import React, { useEffect, useMemo, useState } from 'react'
+import {
+  ClaimMemberStatementInfoFragment,
+  useClaimMemberStatementQuery,
+  useUpdateClaimMemberStatementMutation,
+} from 'types/generated/graphql'
+
+import styled from '@emotion/styled'
+import {
+  Button,
+  CardContent,
+  CardTitle,
+  Spacing,
+  TextArea,
+  Navigation,
+} from '@hedvig-ui'
+import gql from 'graphql-tag'
+import { toast } from 'react-hot-toast'
+import { InfoCircle } from 'react-bootstrap-icons'
+
+const SubmitMemberStatementWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  flex-direction: row;
+`
+
+const SubmitTip = styled(Paragraph)`
+  text-align: right;
+  font-size: 0.8em;
+  color: ${({ theme }) => theme.semiStrongForeground};
+`
+
+gql`
+  query ClaimMemberStatement($claimId: ID!) {
+    claim(id: $claimId) {
+      id
+      memberStatement {
+        ...ClaimMemberStatementInfo
+      }
+    }
+  }
+
+  mutation UpdateClaimMemberStatement(
+    $claimId: ID!
+    $memberStatement: ClaimMemberStatementInput
+  ) {
+    claim_updateMemberStatement(
+      id: $claimId
+      memberStatement: $memberStatement
+    ) {
+      id
+      memberStatement {
+        ...ClaimMemberStatementInfo
+      }
+    }
+  }
+
+  fragment ClaimMemberStatementInfo on ClaimMemberStatement {
+    generalStatement
+    liableParty
+    reasonOfLiability
+  }
+`
+
+const ClaimMemberStatement: React.FC<{
+  claimId: string
+  isEditing: boolean
+  setIsEditing: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
Maybe this is easier to read
```suggestion
  setIsEditing: (boolean) => void
```